tools/ci_build/build.py

```diff
@@ -275,6 +276,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_ENABLE_MICROSOFT_INTERNAL=" + ("ON" if args.enable_msinternal else "OFF"),
                  "-Donnxruntime_USE_BRAINSLICE=" + ("ON" if args.use_brainslice else "OFF"),
                  "-Donnxruntime_USE_NUPHAR=" + ("ON" if args.use_nuphar else "OFF"),
+                 "-Donnxruntime_BUILD_ESSENTIALS=" + ("ON" if args.build_essentials else "OFF"),
```

Comment:
I think this option conflicts with ENABLE_PYTHON. there's a python unit test that tests the profiler, so I think it would fail if both ENABLE_PYTHON and BUILD_ESSENTIALS were both ON.